UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/RenderPipelines/RoughRefractionFeature.cs

```diff
@@ -12,26 +12,66 @@ namespace UnityGLTF
 #endif
 	public class RoughRefractionFeature : ScriptableRendererFeature
 	{
+#if UNITY_2023_1_OR_NEWER
+	    private Downsampling downsampling = Downsampling.None;
+#else
 	    public Downsampling downsampling = Downsampling.None;
+#endif
```

Comment:
Let's not ifdef this and just make it private

Reply:
f092e7eaf927044320db7324037797072454be86